### PR TITLE
[ZEPPELIN-3476] change description "auto-restart interpreter on cron execution"

### PR DESCRIPTION
**Pull request by Savalek**
### What is this PR for?
Description "auto-restart interpreter on cron execution"  is does not quite fit in the sense.

### What type of PR is it?
Documentation

### What is the Jira issue?
[ZEPPELIN-3476](https://issues.apache.org/jira/browse/ZEPPELIN-3476)

### Screenshot
cron_scheduler_dialog_box

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
LGTM

File: docs/usage/other_features/cron_scheduler.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ You can set the cron schedule by filling in this form. Please see [Cron Trigger
 
 You can set the cron executing user by filling in this form and press the enter key.
 
-### auto-restart interpreter on cron execution
+### After execution close the interpreter
```
Comment (felixcheung) [inline docs/usage/other_features/cron_scheduler.md]: maybe "stop" is better than "close"? I think that's the terminology used elsewhere
https://zeppelin.apache.org/docs/0.7.3/manual/interpreters.html#what-is-interpreter-group

**Comment by Savalek** [comment]
@felixcheung, agree with you. Replaced the close on stop.

**Comment by r-kamath** [comment]
LGTM

**Comment by r-kamath** [comment]
will merge this if there are no more discussions

**Comment by Savalek** [comment]
@felixcheung, I did it.

**Pull request closed without merging**